modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -897,6 +899,9 @@ public String getTypeString(Schema p, String prefix, String suffix) {
         if (")".equals(suffix)) {
             fullSuffix = "," + suffix;
         }
+        if (ModelUtils.isAnyTypeSchema(p)) {
+            return "bool, date, datetime, dict, float, int, list, str, none_type";
```

Comment:
We also need to include the prefix before these types and the fullSuffix after them.
Can you update it to this?
```
return prefix + "bool, date, datetime, dict, float, int, list, str, none_type" + fullSuffix;
```
Or you could include ModelUtils.isAnyTypeSchema(p) in the conditionals on lines:
- 906
- 909
And then we wouldn't need this additional if statement because we will correctly return in line 910.
Up to you which way you want to do it.

Reply:
I specifically removed the prefix based on Wing's feedback. prefix/suffix is only used for model, this is done on purpose.

Reply:
> We also need to include the prefix before these types and the fullSuffix after them.
> Can you update it to this?
> 
> ```
> return prefix + "bool, date, datetime, dict, float, int, list, str, none_type" + fullSuffix;
> ```
> 
> Or you could include `ModelUtils.isAnyTypeSchema(p)` in the conditionals on lines:
> 
> * 906
> * 909
> 
> And then we wouldn't need this additional if statement because we will correctly return in line 910.
> Up to you which way you want to do it.

 I think this would make the code more complicate to understand and maintain, I don't think it's worth adding recursion.

Reply:
So reading through the java doc we have:
```
    /**
     * Return a string representation of the Python types for the specified schema.
     * Primitive types in the OAS specification are implemented in Python using the corresponding
     * Python primitive types.
     * Composed types (e.g. allAll, oneOf, anyOf) are represented in Python using list of types.
     *
     * @param p The OAS schema.
     * @param prefix prepended to the returned value.
     * @param suffix appended to the returned value.
     * @return a string representation of the Python types
     */
```
So we see that the prefix and suffix are put on the beginning and the end of the string.
Looking at the call site's javadoc we see that:
```
    /**
     * Output the type declaration of a given name
     *
     * @param p property schema
     * @return a string presentation of the type
     */
    @Override
    public String getTypeDeclaration(Schema p) {
        // this is used to set dataType, which defines a python tuple of classes
        // in Python we will wrap this in () to make it a tuple but here we
        // will omit the parens so the generated documentation will not include
        // them
        return getTypeString(p, "", "");
    }
```
So we see that at the outermost layer we omit setting the prefix and suffix so the documentation works correctly. The prefix and suffix are used for wrapping inner types in the required [], {}, or () characters.

Can't AnyType schemas be inside array items and object properties?
If so, then we need to add the prefix +blah + fullSuffix to include the required [], {}, or () container characters.

Reply:
> So reading through the java doc we have:
> 
> ```
>     /**
>      * Return a string representation of the Python types for the specified schema.
>      * Primitive types in the OAS specification are implemented in Python using the corresponding
>      * Python primitive types.
>      * Composed types (e.g. allAll, oneOf, anyOf) are represented in Python using list of types.
>      *
>      * @param p The OAS schema.
>      * @param prefix prepended to the returned value.
>      * @param suffix appended to the returned value.
>      * @return a string representation of the Python types
>      */
> ```

I've read that but that doc does not actually explain anything. It's not usable documentation that provides actionable information. Good doc provides contextual information. Without context, the prefix could be any arbitrary string. What value can the prefix get? Since the doc does not say anything, you have to assume it can be anything. Is it even ASCII? UTF-8? Is it a python type? Or maybe a package name? or 200 other possibilities.
Just to be clear, I'm adding documentation whenever I can to improve.

Reply:
That doc explains that prefix goes in front, suffix goes behind, and what types they are. I am hearing that you want an explanation of the valid values. Reading the function and my explanation shows that the values can be:
prefix "({[" and empty string
suffix "({[" and empty string
They allow us to enclose the python data types in containing characters which define tuples, dicts, and lists. Does that help?
Thank you for adding the documentation when it helps.
Man I have strived to add a lot of documentation and tests to python-experimental in many many places. When your comments start from a place of this is not a good doc, it is discouraging. Can you give me the benefit of the doubt? I am trying to make improvements; the goal is to inform people and provide a useful tool. This generator has a lot more documentation, features, and tests than a lot of others. I am trying hard here.

Reply:
I think this function should be private or protected. The fact that it's public means it can be invoked from anywhere, which means by contract the caller can pass anything, not just "({["

I understand you mean well, and we are all striving to improve the doc and the code, but also when a particular doc sentence is not helpful, I need to say it so we can improve it. Here are some guidelines for good docs: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html

The doc states this:
@param prefix prepended to the returned value.

From that sentence and the type, you can see the prefix is a string, and the value is just prepended to the return value. But it does not have information about what the prefix is supposed to mean.

Reply:
Sure, I get that.
-This is unclear to me,
-I think that this is missing import information about what characters are allowed
-In the future when you write docs, can you follow this best practice

are all sentences that convey the same intent with starting with this is not good.
I promise you that me and others will respond better with these types of responses. Let's try to treat each other well?

Reply:
Sorry, I was just trying to point out that this particular doc was not helpful. You can see from several of my other PRs that I keep improving the doc, so I don't complain, I just improve the doc.